ios/FluentUI.Demo/FluentUI.Demo.xcodeproj/project.pbxproj

```diff
@@ -7,42 +7,34 @@
-		53A16D7E267C0EE50030FFAC /* AvatarDemoController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 53A16D77267C0EE40030FFAC /* AvatarDemoController.swift */; };
-		53A16D7F267C0EE50030FFAC /* ThemingDemoController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 53A16D78267C0EE40030FFAC /* ThemingDemoController.swift */; };
-		53B659B8257AA44F00070405 /* (null) in Sources */ = {isa = PBXBuildFile; };
-		53B659C4257AA69E00070405 /* DrawerDemoController.swift in Sources */ = {isa = PBXBuildFile; fileRef = A5DCA75D211E3A92005F4CB7 /* DrawerDemoController.swift */; };
-		53B659C5257AA6AA00070405 /* NavigationControllerDemoController.swift in Sources */ = {isa = PBXBuildFile; fileRef = FD41C8F322E28EEB0086F899 /* NavigationControllerDemoController.swift */; };
-		53B659C6257AA6B400070405 /* PopupMenuDemoController.swift in Sources */ = {isa = PBXBuildFile; fileRef = A5961FA8218A61BB00E2A506 /* PopupMenuDemoController.swift */; };
-		53B659C7257AA6C300070405 /* TableViewCellFileAccessoryViewDemoController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 7DC2FB2A24C0F4FD00367A55 /* TableViewCellFileAccessoryViewDemoController.swift */; };
-		53B659C8257AA6CA00070405 /* TooltipDemoController.swift in Sources */ = {isa = PBXBuildFile; fileRef = FD7DF06121FB941400857267 /* TooltipDemoController.swift */; };
-		53B659C9257AA79200070405 /* PassThroughDrawerDemoController.swift in Sources */ = {isa = PBXBuildFile; fileRef = D0F59E3224F4E3A700358DC2 /* PassThroughDrawerDemoController.swift */; };
-		53EA0FA2257ED8AF00899357 /* (null) in Sources */ = {isa = PBXBuildFile; };
-		53F7385F257760B60043071D /* ObjectiveCDemoController.m in Sources */ = {isa = PBXBuildFile; fileRef = FDDD73FB22C6D86B00A9D995 /* ObjectiveCDemoController.m */; };
-		53F7386125776C560043071D /* NotificationViewDemoController.swift in Sources */ = {isa = PBXBuildFile; fileRef = A5B6617523A4227300E801DD /* NotificationViewDemoController.swift */; };
-		53F73862257803DE0043071D /* HUDDemoController.swift in Sources */ = {isa = PBXBuildFile; fileRef = B4E8F65121CD9579008A1598 /* HUDDemoController.swift */; };
-		53F7386B257964AB0043071D /* AvatarGroupViewDemoController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 7D23482924D89C1C00FBE057 /* AvatarGroupViewDemoController.swift */; };
-		53F7386C25796A6D0043071D /* TabBarViewDemoController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 11047D2522932DB1001F0F59 /* TabBarViewDemoController.swift */; };
-		53F7386E25796D730043071D /* DateTimePickerDemoController.swift in Sources */ = {isa = PBXBuildFile; fileRef = FD7254EC21471A3F002F4069 /* DateTimePickerDemoController.swift */; };
+		5303259326B3198A00611D05 /* AvatarDemoController_SwiftUI.swift in Sources */ = {isa = PBXBuildFile; fileRef = 5303259226B3198A00611D05 /* AvatarDemoController_SwiftUI.swift */; };
```

Comment:
what caused these project file changes?

Reply:
Accidentally pulled in some files during merge conflict, fixing that right now so sorry!